### PR TITLE
Add mock support for knowledge graph index

### PR DESCRIPTION
**Pull request by logan-markewich**
Simple PR to add mock support to the knowledge graph index. It should cover using embeddings as well

This was mentioned in https://github.com/jerryjliu/gpt_index/issues/589

### CODE REVIEW

**Review by jerryjliu — COMMENTED** [review]
lgtm! one comment

File: gpt_index/token_counter/mock_chain_wrapper.py [hunk 1]
```diff
@@ -99,5 +109,9 @@ def _predict(self, prompt: Prompt, **prompt_args: Any) -> str:
             return _mock_keyword_extract(prompt_args)
         elif prompt_str == PromptType.QUERY_KEYWORD_EXTRACT:
             return _mock_query_keyword_extract(prompt_args)
+        elif prompt_str == PromptType.KNOWLEDGE_TRIPLET_EXTRACT:
+            return _mock_knowledge_graph_triplet_extract(
+                prompt_args, prompt.partial_dict.get("max_knowledge_triplets", 2)
```
[the remark this answers is not in the document]
Comment (logan-markewich) [reply gpt_index/token_counter/mock_chain_wrapper.py:114]: prompt_args only has the text field for some reason. I had to step through the pdb and find where the max_knowledge_triplets was being stored lol

[the remark this answers is not in the document]
Comment (jerryjliu) [reply gpt_index/token_counter/mock_chain_wrapper.py:114]: ah you're right. it's because we partially format it during initialization.

**Review by jerryjliu — APPROVED** [review]
thanks!!

**Pull request merged**